### PR TITLE
Modify README for the port to MoveIt! 2.0

### PR DESCRIPTION
**Pull request by vmayoral**

### CODE REVIEW

**Comment by rhaschke** [comment]
Dear @vmayoral and @davetcoleman,

I'm not sure who decided to perform MoveIt ROS2 development within a new repository clone. This was not discussed in a maintainer meeting and I don't like the idea, because this makes it much harder to cherry-pick changes back and forth between ROS1 and ROS2 development branches.
I'm not sure, @davetcoleman, you really want to decouple MoveIt ROS2 development completely from ROS1 progress.

**Comment by davetcoleman** [comment]
@rhaschke,

Several reasons led us to this decision:
- Keep PRs related to ROS2 separate from ROS1 so as to not clutter the already large list of pull requests
- We can still rebase and sync the two projects as they have the same git history still, for now. My goal is to keep syncing it for the next few months but eventually it will become more and more challenging and there are some big refactoring changes i want to do that i think will break the ability to sync in the future
- PRs move really slowly in moveit1 and we can't have that with moveit2

File: README.md [hunk 1]
```diff
@@ -1,54 +1,76 @@
...
-build farm | [![Build Status](http://build.ros.org/buildStatus/icon?job=Idev__moveit__ubuntu_trusty_amd64)](http://build.ros.org/job/Idev__moveit__ubuntu_trusty_amd64) | [![Build Status](http://build.ros.org/buildStatus/icon?job=Kdev__moveit__ubuntu_xenial_amd64)](http://build.ros.org/job/Kdev__moveit__ubuntu_xenial_amd64) | [![Build Status](http://build.ros.org/buildStatus/icon?job=Mdev__moveit__ubuntu_bionic_amd64)](http://build.ros.org/job/Mdev__moveit__ubuntu_bionic_amd64) |
+## Continuous Integration Status
 
+[![Build Status](https://travis-ci.org/AcutronicRobotics/moveit2.svg?branch=ros2)](https://travis-ci.org/AcutronicRobotics/moveit2)
 
 ## Docker Containers
 
-[![Docker Build](https://img.shields.io/docker/build/moveit/moveit.svg)](https://hub.docker.com/r/moveit/moveit/builds)
-[![Docker Automated build](https://img.shields.io/docker/automated/moveit/moveit.svg?maxAge=2592000)](https://hub.docker.com/r/moveit/moveit/) [![Docker Pulls](https://img.shields.io/docker/pulls/moveit/moveit.svg?maxAge=2592000)](https://hub.docker.com/r/moveit/moveit/) [![Docker Stars](https://img.shields.io/docker/stars/moveit/moveit.svg)](https://registry.hub.docker.com/moveit/moveit/)
+WIP
```
Comment (davetcoleman) [inline README.md]:
```suggestion
TODO [Create ROS2 Docker containers for MoveIt!](https://github.com/ros-planning/moveit2/issues/15)
```

File: README.md [hunk 2]
```diff
@@ -1,54 +1,76 @@
...
+  - [ ] Create tutorials
 
-service    | Indigo | Kinetic | Melodic
----------- | ------ | ------- | -------
-Travis     | [![Build Status](https://travis-ci.org/ros-planning/moveit.svg?branch=indigo-devel)](https://travis-ci.org/ros-planning/moveit/branches) | [![Build Status](https://travis-ci.org/ros-planning/moveit.svg?branch=kinetic-devel)](https://travis-ci.org/ros-planning/moveit/branches) | [![Build Status](https://travis-ci.org/ros-planning/moveit.svg?branch=melodic-devel)](https://travis-ci.org/ros-planning/moveit/branches) |
-build farm | [![Build Status](http://build.ros.org/buildStatus/icon?job=Idev__moveit__ubuntu_trusty_amd64)](http://build.ros.org/job/Idev__moveit__ubuntu_trusty_amd64) | [![Build Status](http://build.ros.org/buildStatus/icon?job=Kdev__moveit__ubuntu_xenial_amd64)](http://build.ros.org/job/Kdev__moveit__ubuntu_xenial_amd64) | [![Build Status](http://build.ros.org/buildStatus/icon?job=Mdev__moveit__ubuntu_bionic_amd64)](http://build.ros.org/job/Mdev__moveit__ubuntu_bionic_amd64) |
+## Continuous Integration Status
 
+[![Build Status](https://travis-ci.org/AcutronicRobotics/moveit2.svg?branch=ros2)](https://travis-ci.org/AcutronicRobotics/moveit2)
```
Comment (davetcoleman) [inline README.md]: I've just turned on Travis for moveit2:
https://travis-ci.org/ros-planning/moveit2

**Comment by vmayoral** [comment]
Hello @rhaschke,

> This was not discussed in a maintainer meeting and I don't like the idea, because this makes it much harder to cherry-pick changes back and forth between ROS1 and ROS2 development branches.

I'm sorry if we stepped on someone's toes, our intention is to contribute. Our team will be happy to participate in the next developers' session. We started working at https://github.com/AcutronicRobotics/moveit2-original (which should disappear as soon as we stabilize things here, or somewhere else) for the same reasons @davetcoleman provides above, in particular and mainly, to have a clean, easy to manage and accesible (to our team) repository. Overall, to advance faster.

> this makes it much harder to cherry-pick changes back and forth between ROS1 and ROS2 development branches

True, it will become harder but arguably, and as far as I can see for now, the difficulty of cherry-pick*ing* is somewhat _manageable_. Changes from MoveIt! in ROS 1 (for now and while we keep things coherent) shouldn't imply much more than:
```bash
git remote add moveit https://github.com/ros-planning/moveit
git fetch moveit
git cherry-pick ...
```

**Comment by rhaschke** [comment]
@davetcoleman 
> * Keep PRs related to ROS2 separate from ROS1 so as to not clutter the already large list of pull requests

That one valid argument.

> * We can still rebase and sync the two projects as they have the same git history still, for now.

So you started the new repo from the current melodic-branch including all history?
I'm wondering why the latest commits on moveit2's master already contain some unreviewed PRs of mine against ROS1's melodic-devel...

> There are some big refactoring changes i want to do that i think will break the ability to sync in the future

Would be great if you could share your ideas in one of the next maintainer meetings :wink:
Potentially these refactoring changes also might apply to ROS1' MoveIt.

> * PRs move really slowly in moveit1 and we can't have that with moveit2

Do you want to relax the reviewing process to improve on this? If not, I don't see how you can progress faster on MoveIt2 than on MoveIt1. The group of people doing reviews stays the same, doesn't it?

File: README.md [hunk 3]
```diff
@@ -1,54 +1,76 @@
...
-The MoveIt! Motion Planning Framework
+The MoveIt! Motion Planning Framework **for ROS 2.0**
 
-Currently we support ROS Indigo, Kinetic, and Melodic.
+- [Legal disclaimer](#legal-disclaimer)
+- [Milestones](#milestones)
+- [Overview of MoveIt! for ROS 2.0](#)
+- [Installation Instructions](#)
+- [Documentation](#)
+- [Get Involved](#)
```
[the remark this answers is not in the document]
Comment (vmayoral) [reply README.md]: Right, that's a leftover https://github.com/ros-planning/moveit2/pull/1/commits/8b730c8232a7aa4efe92007f52b646eec51c49c6

File: README.md [hunk 4]
```diff
@@ -1,54 +1,76 @@
...
 
-- [Overview of MoveIt!](http://moveit.ros.org)
-- [Installation Instructions](http://moveit.ros.org/install/)
-- [Documentation](http://moveit.ros.org/documentation/)
-- [Get Involved](http://moveit.ros.org/documentation/contributing/)
+## Legal disclaimer
+**WORK IN PROGRESS**
 
-## Continuous Integration Status
+*The material and information contained in this repository is published in good faith and is for the benefit of the general ROS 2.0 community only. You should not rely upon the material or information in this repository as a basis for making any business, legal or any other decisions. Acutronic Robotics makes no representations, claims, promises, guarantees or warranties of any kind, express or implied about the completeness, accuracy, reliability, suitability or adequacy with respect to the software, information or related graphics contained in this repository for any purpose. Acutronic Robotics is not liable for any loss, damage or injury that may arise in connection with this repository. Any action or reliance you take or place based upon the information in this repository is strictly at your own risk.*
```
[the remark this answers is not in the document]
Comment (vmayoral) [reply README.md]: https://github.com/ros-planning/moveit2/pull/1/commits/36bace46cfffaacf7094d253a36644ac098ed236

**Comment by rhaschke** [comment]
Dear @vmayoral 

I appreciate very much your and your team's involvement in migrating MoveIt to ROS2. Thank you very much for this effort in advance! However, I was surprised and confused that we have chosen the approach of a new repo, because we discussed to keep things in sync as long as possible. Now, I'm afraid that the two repos will quickly diverge.

**Comment by vmayoral** [comment]
@rhaschke, thanks for your quick response. 

> Now, I'm afraid that the two repos will quickly diverge.

Well this is certainly concerning. Our team is certainly open to consider other possibilities.
I personally don't see much of an improvement between this approach and having a `ros2` branch within ros-planning/moveit. Quite the opposite, but this is only the feeling of a newcomer.

Maybe having weekly sync-ups with moveit main branch would help @rhaschke?

File: README.md [hunk 5]
```diff
@@ -1,54 +1,76 @@
...
+- [Installation Instructions](#)
+- [Documentation](#)
+- [Get Involved](#)
 
-- [Overview of MoveIt!](http://moveit.ros.org)
-- [Installation Instructions](http://moveit.ros.org/install/)
-- [Documentation](http://moveit.ros.org/documentation/)
-- [Get Involved](http://moveit.ros.org/documentation/contributing/)
+## Legal disclaimer
+**WORK IN PROGRESS**
```
[the remark this answers is not in the document]
Comment (vmayoral) [reply README.md]: Solved in the commit described below.

File: README.md (hunk 2 above)
Comment (vmayoral) [reply README.md]: https://github.com/ros-planning/moveit2/pull/1/commits/f925d83718b1533cebb9196829ca410399d415fc, please review. Note I pointed to `master` branch.

**Comment by rhaschke** [comment]
As you started from MoveIt1's branch, syncing should be indeed feasible.

**Comment by davetcoleman** [comment]
> I'm wondering why the latest commits on moveit2's master already contain some unreviewed PRs of mine against ROS1's melodic-devel...

You already found the source of this issue being a mistaken git push: https://github.com/ros-planning/moveit/issues/1356

Overall, your concerns @rhaschke are all valid, I think we clarified some in an email exchange and we can discuss further at next Thu's maintainer meeting

**Pull request merged**